code/game/objects/structures/musician.dm

```diff
@@ -263,14 +263,12 @@
 	new_tempo = abs(new_tempo)
 	return max(round(new_tempo, world.tick_lag), world.tick_lag)
 // subclass for handheld instruments, like violin
-/datum/song/handheld
 /datum/song/handheld/updateDialog(mob/user)
 	instrumentObj.interact(user)
-/datum/song/handheld/shouldStopPlaying()
+/datum/song/handheld/shouldStopPlaying(mob/user)
 	if(instrumentObj)
```

Comment:
you removed the else clause that would return 1 if `instrumentObj` was null
you also removed the `isliving(instrumentObj.loc)` check
you can replace the entire proc with something like
```
if(!instrumentObj || !isliving(instrumentObj.loc) || user.isUnconscious())
    return TRUE
return FALSE
```
or something like that

Reply:
right Ill fix it

Reply:
Im having a lot of trouble replacing what I have with that, without getting errors

Reply:
The `isliving()` is probably not necessary anyway, since that just checks if you're a `/mob/living`, and you wouldn't be able to be holding it otherwise
It should probably check for `user == loc` instead
That would also keep you from handing it to someone else and continuing to play it, which... it looks like you can do right now, but I'm not sure